packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -182,10 +182,245 @@ export interface LocalSecondaryIndexProps extends SecondaryIndexProps {
   readonly sortKey: Attribute;
 }
 
+/**
+ * Imported or created dynamodb table
+ */
+export interface ITable extends IResource {
+  /**
+   * arn of the dynamodb table
```

Comment:
Docstrings should be proper sentences (start with a capital letter, end in a period).

Same comment for all the other docstrings in this file.